sandbox-web/src/main/java/com/exadel/sandbox/team5/error/RestExceptionHandler.java

```diff
@@ -0,0 +1,52 @@
+package com.exadel.sandbox.team5.error;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+import org.springframework.web.servlet.NoHandlerFoundException;
+
+import java.util.NoSuchElementException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+
+@ControllerAdvice
+public class RestExceptionHandler {
+    private static Logger logger = Logger.getLogger("RestExceptionHandler");
```

Comment:
logger should be final and it will be better to use ``` Logger.getLogger(RestExceptionHandler.class.getCanonicalName())```. Or any lombok annotation https://projectlombok.org/features/log

Reply:
I will use @Slf4j